### ISSUE TITLE
Seeddata help popup

### ISSUE THREAD

**Issue by ashokkrish** [ashokkrish/spatialEpisim#58]
@bryce-carson 

When the user selects the "?" 
<img width="356" alt="image" src="https://github.com/user-attachments/assets/c63b41e9-a5fe-42fc-a289-84daa24ab3ee">

The following popup appears

<img width="596" alt="image" src="https://github.com/user-attachments/assets/f825bc65-382f-4426-8678-69d138ee2119">

Could you show the COD_InitialSeedData.csv data as opposed what is currently shown please?

I want to see: 
 
<img width="704" alt="image" src="https://github.com/user-attachments/assets/13eea672-9154-4141-8569-f114085e2faa">

**Comment by bryce-carson** [comment]
@ashokkrish, yep! Working on that. I created a fix for the helper for the aggregation factor first. I also fixed some small typographical issues in the lambda helper and the psi helper.

**Comment by bryce-carson** [comment]
@ashokkrish, all done! Please try the new code and UI.
